dist/bom/controlplane-base-bom/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
     api(project(":core:common:edr-store-core"))
     api(project(":core:common:runtime-core"))
     api(project(":core:common:token-core"))
+    api(project(":core:common:runtime-core"))
```

Comment:
wait, it was already part of the bom!

Reply:
huh? i didn't see it there... will revert.